### ISSUE TITLE
Deprecate jsx closing tag rule in favor of react/jsx-tag-spacing rule

### ISSUE THREAD

**Issue by cyberhck** [crazyfactory/tslint-rules#44]
closes #14 

The rule supports multiple convention AND has a fixer for those. Let's use that and introduce that in boilerplate as well :)

here's the link:
https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-tag-spacing.md

what do you think @paibamboo

**Comment by paibamboo** [comment]
Yes, I think we are using that [rule](https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-tag-spacing.md) in CF projects.
